app/routers/vip.py

```
@@ -1,7 +1,7 @@
 import logging
 import os
 import inspect
-from typing import Any, Callable, Dict, Literal, Optional, Type, Union, cast
+from typing import TYPE_CHECKING, Any, Callable, Dict, Literal, Optional, Type, Union, cast
```

Flake8 (7.3.0)

[error] 'typing.TYPE_CHECKING' imported but unused

(F401)